STYLE_API_TEMPLATE.md

```diff
@@ -22,15 +24,17 @@ See also [Examples](https://alpha-docs-aws.amazon.com/awsstyleguide/latest/style
 
 ## Basic elements for documentation
 
-The following sections describe the basic API documentation structure. Each section is discussed under its respective heading below. Include those elements appropriate to the API. Depending on where the documentation appears within a section or subsection, heading levels may be adjusted to fit with other content.
+The following sections describe the basic API documentation structure. Each section is discussed under its respective heading below. You can include only those elements appropriate to the API. 
+
+Depending on where the documentation appears within a section or subsection, heading levels may be adjusted to fit with other content.
 
 1. Name of API (heading level 2)
 2. Path parameters (heading level 3)
 3. Query parameters (heading level 3)
-4. Sample request (heading level 4)
-5. Sample response (heading level 4)
-6. Request body (heading level 3)
-7. Sample request (heading level 4)
+4. Request fields (heading level 3)
+5. Response fields (heading level 3)
+6. Sample request (heading level 4)
+7. Sample response (heading level 4)
 
```

Comment:
Could we switch the order so that request and response headings are grouped? The new order will be:
Name of API
Path parameters
Query parameters
Request fields
Sample request
Response fields
Sample response.